stack.yaml

```diff
@@ -1,8 +1,10 @@
+# https://www.stackage.org/lts-13.27
 resolver: lts-13.27
 
 packages:
 - ./hs-abci-server
 - ./hs-abci-extra
+- ./hs-tendermint
```

Comment:
I think we should change the package name to `hs-tendermint-client` and use the `Client`  path in all the modules